pkg/controller/cincinnati/new.go

```diff
@@ -318,6 +316,26 @@ func (k *kubeResources) newDeployment(instance *cv1alpha1.Cincinnati) *appsv1.De
 		}
 	}
 
+	if instance.Spec.CmKey != "" {
+		v := corev1.Volume{
+			Name: "trusted-ca",
```

Comment:
Should make `"trusted-ca"`, at a minimum, a package local variable so we don't mess it up w.r.t. the container's volume mount.